src/Curl.Converter.Net/Curl.Converter.Net.csproj

```diff
@@ -0,0 +1,52 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.1;netcoreapp3.1;net5.0;net6.0</TargetFrameworks>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <PackageId>Curl.Converter.Net</PackageId>
+    <Authors>curl-to-csharp</Authors>
+    <Description>
+      A cURL parser and C# code converter based on curl-to-csharp project for .NET.
+
+      Supported platforms:
```

Comment:
Let's remove the supported platforms.
NuGet automatically detects them based on the TargetFrameworks

Reply:
Removed